diff --git a/inc/config.php b/inc/config.php
index 9c05092..d5de0e3 100644
--- a/inc/config.php
+++ b/inc/config.php
@@ -9,6 +9,8 @@ class Config {
        public static $packageDir;
 }
 
-Config::$dbName = 'sqlite:../db/packages.sqlite3';
+Config::$dbName = 'mysql:host=mysql;dbname=nuget';
+Config::$dbUsername = 'nuget';
+Config::$dbPassword = '{{ password }}';
 Config::$packageDir = __DIR__ . '/../packagefiles/';
 Config::$apiKey = 'ChangeThisKey';
